### PR TITLE
Dedicated user event classes

### PR DESCRIPTION
**Pull request by szymach**

### CODE REVIEW

**Comment by szymach** [comment]
@fsi-open

File: Command/CreateUserCommand.php [hunk 1]
```diff
@@ -108,7 +108,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         if (true === $input->getOption('enforce-password-change')) {
             $user->enforcePasswordChange(true);
         }
-        $this->eventDispatcher->dispatch(new UserEvent($user), AdminSecurityEvents::USER_CREATED);
+        $this->eventDispatcher->dispatch(new UserCreatedEvent($user), AdminSecurityEvents::USER_CREATED);
```
Comment (chives) [inline Command/CreateUserCommand.php:111]: There is no need for the second argument. We removed it everywhere in `admin-bundle`

**Review by chives — COMMENTED** [review]
Shouldn't we replace symfony's `EventDispatcherInterface` with the one from psr-14?

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open

**Review by rn0 — APPROVED** [review]

**Review by chives — APPROVED** [review]

**Pull request merged**